src/workspaces/common/state/useAppPolling.ts

```diff
@@ -28,17 +27,15 @@ export const useAppPolling = (name: string, namespace: string, workspace?: Works
   };
   const loadApps = async (maybeStale?: boolean): Promise<void> => {
     try {
-      const newGoogleApps =
-        workspace?.workspaceInitialized && isGoogleWorkspace(workspace)
```

Comment:
`workspaceInitialized` looks like it's related to storage, so I don't think it's necessary here. _If_ it is, I would suggest that the `Wrapper` component not render its children until the workspace is initialized.